charmcraft.yaml

```diff
@@ -13,17 +13,8 @@ bases:
       channel: "22.04"
 parts:
   charm:
-    build-packages:
-    - build-essential
-    - cargo
-    - libffi-dev
-    - libssl-dev
-    - libxml2
-    - libxslt1-dev
-    - pkg-config
-    - python3-lxml
-    - python3-openssl
-    - python3-pip
-    - python3-setuptools
-    - python3-wheel
-    - rustc
+    charm-binary-python-packages:
+    - lxml==5.2.1
```

Comment:
This is duplicated inn `requirements-binary.txt`. You should probably refer here to the file instead

Reply:
Yes, indeed, I added a comment and also added a test to make sure they are in sync, thanks!

Reply:
Why not using the file here instead?